Gruntfile.js

```diff
@@ -264,7 +264,7 @@ module.exports = function(grunt) {
 					fixture: 'test/runner.tmpl',
 					testCwd: 'test/core',
 					data: {
-						title: 'aXe Core Tests'
+						title: 'axe Core Tests'
```

Comment:
Nitpick: Should we capitalize here (and all other places in the file where it starts the `title`?

Reply:
Yep, makes sense to me!